tests/runtime/utils/test_servicer_util.py

```diff
@@ -68,7 +68,7 @@ def test_servicer_util_validate_caikit_library_class_exists_returns_caikit_class
 
 def test_servicer_util_validate_caikit_library_class_exists_raises_for_garbage_input():
     """Test that validate_caikit_library_class_exists raises for garbage 'model'"""
-    with pytest.raises(AttributeError):
+    with pytest.raises(ValueError):
```

Comment:
I don't understand how your changes made this now be a value error? 🤷 

But I'm okay with it if we're discussing removing these checks anyway